### PR TITLE
add np_ method

### PR DESCRIPTION
**Pull request by rgraff**

### CODE REVIEW

File: lib/fast_gettext/translation.rb [hunk 1]
```diff
@@ -72,6 +72,12 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+
+    def np_(context, key, *args, &block)
+      options = (args.last.is_a? Hash) ? args.pop : {}
```
Comment (rgraff) [inline lib/fast_gettext/translation.rb]: This works differently the other methods for passing in the separator. Will note the difference in the readme.

File: lib/fast_gettext/translation.rb [hunk 2]
```diff
@@ -72,6 +72,12 @@ def ns_(*args, &block)
...
+      nargs = ["#{context}#{options[:separator]||CONTEXT_SEPARATOR}#{key}"] + args
+      n_(*nargs){nil} or (block ? block.call : key)
```
Comment (grosser) [inline lib/fast_gettext/translation.rb]: why not pass the block to n_ ?

Comment (grosser) [reply lib/fast_gettext/translation.rb]: ah nvm ... got it ...

File: spec/fast_gettext/translation_spec.rb [hunk 3]
```diff
@@ -151,6 +151,17 @@
...
   end
 
+  describe :np_ do
+    it "translates whith namespace" do
+      np_('Fruit','Apple','Apples',2).should == 'Apples'
+    end
+
+    it "returns block when specified" do
+      np_('not','found'){:block}.should == :block
+      np_('not','found'){nil}.should be_nil
```
Comment (grosser) [inline spec/fast_gettext/translation_spec.rb]: test for not-found without block

**Comment by rgraff** [comment]
@grosser this one should be good to go too

**Pull request merged**